ocaml/xapi/create_misc.ml

```diff
@@ -89,6 +89,14 @@ let make_kpatch_list () =
 
 open Xstringext
 
+(** [count_cpus] returns the number of CPUs found in /proc/cpuinfo *)
+let count_cpus () =
+  let cpuinfo = "/proc/cpuinfo" in
+  let re = Re_perl.compile @@ Re_perl.re {|^processor\s*:\s+\d+|} in
```

Comment:
I wasn't sure whether this'd work on older versions of OCaml, but according to https://caml.inria.fr/pub/docs/manual-ocaml/extn.html `{|quoted strings|}` were introduced in OCaml 4.02 so should be fine.

Reply:
We should really get travis on this branch...